androidcomponents/androidcomponents.go

```diff
@@ -58,12 +48,14 @@ func InstallLicences(androidSdk *sdk.Model) error {
 	}
 
 	if !sdkManager.IsLegacySDK() {
-		licensesCmd := command.New(filepath.Join(androidSdk.GetAndroidHome(), "tools/bin/sdkmanager"), "--licenses")
-		licensesCmd.SetStdin(bytes.NewReader([]byte(strings.Repeat("y\n", 1000))))
+		cmdOpts := command.Opts{
+			Stdin: bytes.NewReader([]byte(strings.Repeat("y\n", 1000))),
+		}
+		licensesCmd := cmdFactory.Create(filepath.Join(androidSdk.GetAndroidHome(), "tools/bin/sdkmanager"), []string{"--licenses"}, &cmdOpts)
```

Comment:
Can you please update this to useandroidSdk. CmdlineToolsPath(), instead of tools/bin string hardcoded?